lib/service/formula_wrapper.rb

```diff
@@ -10,9 +11,10 @@ class FormulaWrapper
     # Create a new `Service` instance from either a path or label.
     def self.from(path_or_label)
       return unless path_or_label =~ path_or_label_regex
+      return unless (formula = Regexp.last_match(1)).present?
 
       begin
-        new(Formulary.factory(Regexp.last_match(1)))
+        new(Formulary.factory(formula))
```

Comment:
Probably worth `T.must`ing this one so that it throws an error if it's missing (which should be programmer error as if the regex matches but is missing a group then the regex is wrong).